python/tank_vendor/shotgun_authentication/console_authentication.py

```diff
@@ -103,6 +103,16 @@ def _get_password(self):
             raise AuthenticationCancelled()
         return password
 
+    def _raw_input(self, text):
+        """
+        Wraps the raw_input builtin function so unit tests can mock it.
+
+        :param text: Text to display before prompting the user.
+
+        :returns: The user's text input.
+        """
+        return raw_input(text)
```

Comment:
Very minor, but if there's never a case where we want to preserve leading/trailing whitespace then we could potentially do the strip here and not have to worry about it elsewhere.